certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -969,6 +991,19 @@ PROCEDURE READCONFIGFILESETTINGS
    FILECLOSE(LFNUMBER,LFERROR)
   END
 
+[* Update debugmode
+**   If the "DB" parameter is set to false, then force debug
+**     mode to off (overrides all other debug mode logic settings).
+**   If the "DB" parameter is set to true or the installation date was found and
+**     less than 90 days prior, set the debug mode to true.
+*]
+ IF PARAMDEBUGMODE=BNODEBUGMODEOFF THEN
+  BNODEBUGMODE=BNODEBUGMODEOFF
+ ELSE IF PARAMDEBUGMODE=BNODEBUGMODEON OR
+     (PARAMINSTALLDATE<>DATENULL AND
+      PARAMINSTALLDATE+90>SYSTEMDATE) THEN
```

Comment:
at line 1004 I suggest using BNOPRINTDEBUGINFODAYS instead of 90 in "PARAMINSTALLDATE+90>SYSTEMDATE) THEN"

Reply:
@jordan-keenan changed